### ISSUE TITLE
WebSocket error 502 on mybinder.org

### ISSUE THREAD

**Issue by TheBear44** [jupyterlab/jupyterlab-demo#82]
I'm getting this result from chrome console on mybinder.org when notebook is trying to connect to kernel.
```
WebSocket connection to 'wss://hub.mybinder.org/user/jupyterlab-jupyterlab-demo-azimq5mq/api/kernels/267f4698-ce2a-443d-aab9-46bd61a49ffb/channels?session_id=8d0e4978-7cb6-4184-82b3-4bf6fa323698&token=Qy1Nd1A6RDWoJEL5lUqo_w' failed: Error during WebSocket handshake: Unexpected response code: 502
```
Meanwhile, the top right status dot keeps showing `Kernel Reconnecting`.

When I visit the normal notebook (path: `/notebooks/demo/Lorenz.ipynb`), `500 : Internal Server Error` is returned.

**Comment by yuvipanda** [comment]
Looking at the logs, I see:

```
[E 21:38:48.121 NotebookApp] Uncaught exception GET /user/jupyterlab-jupyterlab-demo-sd115y5w/notebooks/demo/Lorenz.ipynb (162.238.29.51)
    HTTPServerRequest(protocol='https', host='hub.mybinder.org', method='GET', uri='/user/jupyterlab-jupyterlab-demo-sd115y5w/notebooks/demo/Lorenz.ipynb', version='HTTP/1.1', remote_ip='162.238.29.51')
    Traceback (most recent call last):
      File "/srv/conda/lib/python3.6/site-packages/tornado/web.py", line 1697, in _execute
        result = method(*self.path_args, **self.path_kwargs)
      File "/srv/conda/lib/python3.6/site-packages/tornado/web.py", line 3174, in wrapper
        return method(self, *args, **kwargs)
      File "/srv/conda/lib/python3.6/site-packages/notebook/notebook/handlers.py", line 59, in get
        get_custom_frontend_exporters=get_custom_frontend_exporters
      File "/srv/conda/lib/python3.6/site-packages/notebook/base/handlers.py", line 463, in render_template
        return template.render(**ns)
      File "/srv/conda/lib/python3.6/site-packages/jinja2/asyncsupport.py", line 76, in render
        return original_render(self, *args, **kwargs)
      File "/srv/conda/lib/python3.6/site-packages/jinja2/environment.py", line 1008, in render
        return self.environment.handle_exception(exc_info, True)
      File "/srv/conda/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception
        reraise(exc_type, exc_value, tb)
      File "/srv/conda/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise
        raise value.with_traceback(tb)
      File "/srv/conda/lib/python3.6/site-packages/notebook/templates/notebook.html", line 1, in top-level template code
        {% extends "page.html" %}
      File "/etc/jupyter/templates/page.html", line 1, in top-level template code
        {% extends "templates/page.html" %}
      File "/srv/conda/lib/python3.6/site-packages/notebook/templates/page.html", line 154, in top-level template code
        {% block header %}
      File "/srv/conda/lib/python3.6/site-packages/notebook/templates/notebook.html", line 120, in block "header"
        {% for exporter in get_custom_frontend_exporters() %}
      File "/srv/conda/lib/python3.6/site-packages/notebook/notebook/handlers.py", line 19, in get_custom_frontend_exporters
        from nbconvert.exporters.base import get_export_names, get_exporter
      File "/srv/conda/lib/python3.6/site-packages/nbconvert/__init__.py", line 7, in <module>
        from . import postprocessors
      File "/srv/conda/lib/python3.6/site-packages/nbconvert/postprocessors/__init__.py", line 5, in <module>
        from .serve import ServePostProcessor
      File "/srv/conda/lib/python3.6/site-packages/nbconvert/postprocessors/serve.py", line 19, in <module>
        class ProxyHandler(web.RequestHandler):
      File "/srv/conda/lib/python3.6/site-packages/nbconvert/postprocessors/serve.py", line 21, in ProxyHandler
        @web.asynchronous
    AttributeError: module 'tornado.web' has no attribute 'asynchronous'
```

**Comment by ian-r-rose** [comment]
Fixed by #83, thanks @TheBear44!
